test/wdio/setup.ts

```diff
@@ -13,7 +13,8 @@ const testRequiresManualSetup =
   window.__wdioSpec__.includes('custom-elements-output-tag-class-different') ||
   window.__wdioSpec__.includes('custom-elements-delegates-focus') ||
   window.__wdioSpec__.includes('custom-elements-output') ||
-  window.__wdioSpec__.includes('global-script');
+  window.__wdioSpec__.includes('global-script') ||
+  window.__wdioSpec__.endsWith('custom-tag-name.test.tsx');
```

Comment:
we need this because we don't want the normal `.define` call to happen, right?

Reply:
Yes, so we avoid registering all the components to the registry to test the behavior where user would do this manually.